api/v1beta2/foundationdb_database_configuration.go

```diff
@@ -710,6 +727,8 @@ const (
 	RedundancyModeDouble RedundancyMode = "double"
 	// RedundancyModeTriple defines the replication factor 3.
 	RedundancyModeTriple RedundancyMode = "triple"
+	// RedundancyModeThreeDataHall defines the replication factor three_data_hall.
+	RedundancyModeThreeDataHall RedundancyMode = "three_data_hall"
```

Comment:
Make a note that `three_data_hall` is not tested in the combination with multi-region.